pages/people/[id]/details.tsx

```diff
@@ -4,17 +4,40 @@ import { GetServerSideProps } from 'next';
 import { Resident } from 'types';
 import PersonDetails from 'components/PersonView/PersonDetails';
 import { isAuthorised } from '../../../utils/auth';
+import { useRouter } from 'next/router';
+import { useState, useEffect } from 'react';
+import ConfirmationBanner from 'components/ConfirmationBanner/ConfirmationBanner';
+import { ConditionalFeature } from 'lib/feature-flags/feature-flags';
 
 interface Props {
   person: Resident;
 }
 
-const PersonAllocationsPage = ({ person }: Props): React.ReactElement => (
-  <Layout person={person}>
-    <PersonDetails person={person} />
-  </Layout>
-);
+const PersonAllocationsPage = ({ person }: Props): React.ReactElement => {
+  const [showConfirmation, setShowConfirmation] = useState(false);
+  const router = useRouter();
 
+  useEffect(() => {
+    setShowConfirmation(
+      Boolean(router.query.flaggedStatus) && person.contextFlag === 'C'
+    );
+  });
+
+  return (
+    <>
+      {showConfirmation && (
+        <ConditionalFeature name="case-status">
+          <ConfirmationBanner title={'Flagged status added'}>
+            <></>
+          </ConfirmationBanner>
```

Comment:
```suggestion
          <ConfirmationBanner title={'Flagged status added'} />
```
I think we should be able to do this, if a component doesn't have children.

Reply:
Fixed!

This was because it was holding a link before (which I've removed for now as it's gonna be there in the next iteration)